### ISSUE TITLE
Draft Facility Locator mobile map research plan

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#14515]
### User story
**AS A** Researcher
**I WANT** to document the details for planning Facility locator mobile map research
**SO THAT** the research will achieve its purpose

### Templates
Use the appropriate template for the group you're conducting research with:
~~- [Editor-facing Research Plan Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/research/sitewide-cms-editor-research-plan-template.md)~~
- [Veteran-facing Research Plan Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/research-plan-template.md)

## Suggested workflow/tasks (Sprint 99):
- [x] Brainstorm ideas for research
- [x] Create initial draft of research plan
- [x] Create post in Slack tagging UX Lead and PO. UX Lead will provide consolidated feedback in the Slack thread
- [x] Solicit review/feedback from Product Manager (PM) and/or other team members

## Suggested workflow/tasks (Sprint 100):
- [x] Review feedback from UX lead and others
- [x] Follow-up with feedback providers if there are questions
- [x] Make changes
- [x] Document insights relevant to future phases of the project

## Suggested workflow/tasks (blocked on coded prototype being built and the conversation guide):
- [ ] Identify moderator(s), notetaker(s), and observer(s) if moderated research
- [ ] Identify time slots for research sessions (twice as many slots as # of participants desired) if moderated research
- [ ] Post on github in project research folder for final approval before submitting to Research Review cycle

## Acceptance Criteria (Sprint 99) - 8 points
- [x] Research plan drafted

## Acceptance Criteria (Sprint 100) - 8 points
- [x] Revise draft based on UX Lead and other feedback

## ~~Acceptance Criteria (blocked on coded prototype being built and the conversation guide)~~ 
- [ ] ~~Research plan posted on github in project research folder~~
- [ ] ~~Final approval obtained from Sitewide UX Lead and documented at the bottom of the file in preparation for Research Review cycle~~
Relocated to future ticket: https://dsva.slack.com/archives/C0FQSS30V/p1704302907385019

**Comment by xiongjaneg** [comment]
The ticket description and ACS will need refining (or a new ticket may be substituted) based upon the updated research ticket templates in #15829

**Comment by jilladams** [comment]
FYSA: per Cindy's mid-sprint checkin today, #15829 may be delayed into S97.

**Comment by xiongjaneg** [comment]
currently, conversation guides are a separate template

**Comment by ALogsdon3** [comment]
AC for sprint 99 complete. Moved 2nd AC to sprint 100, per @xiongjaneg 
I want to note that I don't foresee the last two ACs on sprint 100 being completed next sprint, since they are dependent on both a coded prototype being built and the conversation guide being completed. For clarity's sake, those would be:
- [ ] Research plan posted on github in project research folder
- [ ] Final approval obtained from Sitewide UX Lead and documented at the bottom of the file in preparation for Research Review cycle

**Comment by jilladams** [comment]
@xiongjaneg @ALogsdon3 @aklausmeier (cc @mmiddaugh ): Based on Alexis' note above I went ahead and updated the Sprint 100 ACs, but please modify if you all feel that's incorrect.

**Comment by davidmpickett** [comment]
- [Draft plan in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1701208273582/c0f537ca54bc4004f6a33473d3302dc251a7b05e?wid=1-1702498212284&sender=u9c899abc4fe36c9a698b1647)
- [Slack thread for feedback](https://dsva.slack.com/archives/C0FQSS30V/p1702586999338709)

**Comment by jilladams** [comment]
Acknowledged in planning: @ALogsdon3 will have a very minimal Sprint 100. We've pulled this into sprint with the understanding that when @ALogsdon3 is back online, we'll regroup with @aklausmeier just to work out exactly what'll be doable with those couple of days, after the research plan updates to incorporate feedback. No expectation that this ticket will make substantive progress in current sprint, til we have that discussion.

**Comment by aklausmeier** [comment]
[Consolidated feedback](https://dsva.slack.com/archives/C0FQSS30V/p1703206215488309?thread_ts=1702586999.338709&cid=C0FQSS30V) provided in Slack thread

**Comment by ALogsdon3** [comment]
Today I reviewed @aklausmeier's feedback and then met with her to ask follow up questions. I also reviewed @davidmpickett's comments in Mural. 
I incorporated all the feedback that was actionable at the moment, and will record outstanding issues in a separate comment.

**Comment by ALogsdon3** [comment]
@davidmpickett asked: "Do we want to have something specifically addressing AT. Like "Veterans using assistive technology will be able to interact with the map in functionally equivalent ways to those not using assistive technology." or something?"
Based on my conversation with Amanda and after reviewing existing studies and guidelines, I updated the research questions and hypotheses to make it clear that we wanted to know how our prototype is working for AT users. It's important not to alter the tasks or research questions based on the way a person is accessing the prototype. But hopefully the updated wording is inclusive enough to account for screen reader-mediated use. 

Dave also said: "The current designs don't show a facility operating status. This is another key element of search results that should be included in testing."
I'm not sure whether this is a comment about an absence in the research plan or in the prototype itself. Can you clarify? 

I made several revisions to the research plan based on both Amanda and Dave's suggestions and questions. A few that I didn't make at this time:
* updates to recruitment criteria. The header/footer research that Amanda and Cindy are undertaking will inform how we recruit for this study. We will leverage lessons learned from recruiting AT users via Perigean. It doesn't make sense to flesh this out in detail if there's an opportunity to replicate what works with header/footer study.
* recommendations related to the conversation guide. I will incorporate the convo guide feedback when I begin working on that in sprint 101.

**Comment by ALogsdon3** [comment]
One more question from @davidmpickett that I forgot to address with @aklausmeier today and think maybe @thejordanwood should be looped in on:
"Which facility types will we be testing? The design and these questions seem focused mostly on VAMCs.
Do we want to also include tasks that cover Vet Center, VBA, NCA, etc?" 

My instinct is that yes, we should have questions related to different facility types, if only to prompt participants to interact with the filters. But I'm unclear whether there are other reasons to consider including other facility types.

**Comment by davidmpickett** [comment]
> Dave also said: "The current designs don't show a facility operating status. This is another key element of search results that should be included in testing."
> I'm not sure whether this is a comment about an absence in the research plan or in the prototype itself. Can you clarify?

Both? Neither? 😅

Facility operating status might not rise to the level of being a Research Plan consideration (since nothing about it is changing for this iteration), but wanted to flag its conspicuous absence from the Sketch designs as you move into Conversation Guide and prototype discussions. It's an element of the existing Facility Locator, so unless there was a reason to specifically exclude it, I'd expect to see it somewhere in the prototype. 

![Screenshot 2024-01-02 113859](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/483a6663-f365-4525-8714-61ae469c5c5e)

![Screenshot 2024-01-02 113912](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/43ea1700-66c5-43aa-8a49-d4eab7165049)

![Screenshot 2024-01-02 113958](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/72c34de8-dc80-41b0-bc89-6d4025712089)

**Comment by davidmpickett** [comment]
> One more question from @davidmpickett that I forgot to address with @aklausmeier today and think maybe @thejordanwood should be looped in on: "Which facility types will we be testing? The design and these questions seem focused mostly on VAMCs. Do we want to also include tasks that cover Vet Center, VBA, NCA, etc?"
> 
> My instinct is that yes, we should have questions related to different facility types, if only to prompt participants to interact with the filters. But I'm unclear whether there are other reasons to consider including other facility types.

There are also a couple subtle differences in the results listings for some facility types:
- Urgent care and Emergency care results have an alert about in-network benefits
- Community providers results tend to be people's names rather than building/company names

Again, not huge differences, but more just wanting to make sure we're not overlooking any aspects of the product that might be fruitful to include in interaction testing

![Screenshot 2024-01-02 114745](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/28c5a0b6-2cba-46ac-a0ab-38005912a763)

![Screenshot 2024-01-02 114658](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/8e09abeb-24c2-4f76-8587-1a3d8e9c82ea)

**Comment by ALogsdon3** [comment]
Thanks, @davidmpickett. I will keep all of this in mind as I work on the conversation guide. 
Flagging this for @thejordanwood as well. 
The results will be numbered instead of lettered, and the distance will appear under the facility or provider name instead of next to the number of the result, but I don't know that any of the rest of the content will show up differently from how it does now. But it does seem important to acknowledge that some of what we'll see in a prod prototype hasn't been presented in the designs so far.

**Comment by ALogsdon3** [comment]
@jilladams @xiongjaneg all the ACs for sprint 100 have been completed.

**Comment by xiongjaneg** [comment]
@xiongjaneg break remaining AC work in to a new ticket

**Comment by ALogsdon3** [comment]
[Link to research plan in github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/user-research/2024-01-mobile-map/conversation-guide.md)

**Comment by aklausmeier** [comment]
@ALogsdon3 updated your link from convo guide to research plan

**Comment by aklausmeier** [comment]
@xiongjaneg since this is being left in a "draft" mode, should we spin up a ticket in the backlog to "finalize"

**Comment by xiongjaneg** [comment]
@aklausmeier That would be great. Is there a template for that? If not, I can stub on off of this ticket.

**Comment by aklausmeier** [comment]
@xiongjaneg probably best to stub off these tickets so we know what was done/still needs to be done

**Comment by xiongjaneg** [comment]
Stubbed follow-up #16924
